demos/supabase/config/sync_rules.yaml

```diff
@@ -0,0 +1,11 @@
+# See Documentation for more information:
+# https://docs.powersync.com/usage/sync-rules
+# Note that changes to this file are not watched.
+# The service needs to be restarted for changes to take effect.
+bucket_definitions:
+  user_lists:
+    # Separate bucket per todo list
+    parameters: select id as list_id from lists where owner_id = token_parameters.user_id
```

Comment:
Can we use the latest syntax for `token_parameters.user_id` (released yesterday)? It should be `request.user_id()` - see details here: https://docs.powersync.com/usage/sync-rules/parameter-queries

Reply:
I updated this now. 

I noticed I had to explicitly pull the latest image with `docker pull journeyapps/powersync-service` : since this feature is only available in the latest version and I had already started this demo before.

I think first time runs should not need to do this though.